test/common/network/cidr_range_test.cc

```diff
@@ -215,10 +207,10 @@ TEST(Ipv6CidrRange, StringCtor) {
   EXPECT_EQ(rng.asString(), "::fc00/118");
   EXPECT_EQ(rng.length(), 118);
   EXPECT_EQ(rng.version(), IpVersion::v6);
-  EXPECT_FALSE(rng.isInRange(Utility::parseInternetAddress("::fbff")));
-  EXPECT_TRUE(rng.isInRange(Utility::parseInternetAddress("::fc00")));
-  EXPECT_TRUE(rng.isInRange(Utility::parseInternetAddress("::ffff")));
-  EXPECT_FALSE(rng.isInRange(Utility::parseInternetAddress("::1:00")));
+  EXPECT_FALSE(rng.isInRange(Ipv6Instance("::fbff")));
+  EXPECT_TRUE(rng.isInRange(Ipv6Instance("::fc00")));
+  EXPECT_TRUE(rng.isInRange(Ipv6Instance("::ffff")));
+  EXPECT_FALSE(rng.isInRange(Ipv6Instance("::1:00")));
```

Comment:
Do these provide 100% coverage of `isinRange`? There are quite a few different branches there now..